assets/scss/base/_table.scss

```diff
@@ -120,3 +120,7 @@ table {
     float: left;
   }
 }
+
+.table--font-small {
```

Comment:
> I was modelling it on our font sizes in isolation, but I think you're right. If the base font of the table component is 16px, then I should probably call this selector table--medium or table--large (I left out the word font as thought it's cleaner and it'd be pretty obvious what is mean,t but happy to hear your thoughts on this).

I'd be tempted with or `.table--reset`
If it is just the `p` element (the only element with `core-16` on it in `table`) go with the below
```
.table--reset {
    p {
        @include core-19;
    }
}
```